src/ralph/backends/http/lrs.py

```diff
@@ -4,25 +4,47 @@
 from ralph.backends.http.async_lrs import AsyncLRSHTTP
 
 
+def _check_for_running_loop(func):
+    """Raise an error when methods are used in a running asyncio events loop."""
+
+    def wrap(*args, **kwargs):
+        """Wrapper for decorator function."""
+        try:
+            loop = asyncio.get_running_loop()
+        except RuntimeError:
+            return func(*args, **kwargs)
+        if loop.is_running():
+            raise RuntimeError(
+                f"An event loop is already running. You must use "
```

Comment:
```suggestion
                f"This event loop is already running. You must use "
```

Reply:
Commited this a bit too quickly. 

Reverted to "This event loop [..;]" to be consistent with the syntax of the original error raised by `asyncio`.